dataset.proto

```diff
@@ -34,9 +40,11 @@ message DataSet {
     string url = 1;
   }
 
-  Categorization categorization = 1;
-  AdFraud        adfraud        = 2;
-  // 3 and 4 are reserved
-  Malicious      malicious      = 5;
-  Echo           echo           = 6;
+  Categorization         categorization = 1;
+  AdFraud                adfraud        = 2;
+  // 3 is reserved
+  // if SENTIMENT is not requested, keywords will map to zero values  
```

Comment:
No need for "if" logic at this level, really, as it's a DataSet.  It needs to be simple.